### PR TITLE
Add OWASP IDE VulScanner

### PR DESCRIPTION
**Pull request by Jitu-Ranjan**
Add OWASP IDE VulScanner

### CODE REVIEW

File: _data/tools.json [hunk 1]
```diff
@@ -1934,5 +1934,14 @@
       "platforms": "Linux, MacOS, Windows",
       "note": "A vulnerability scanner for container images and filesystems.",
       "type": "SAST"
+   },
+    {
+      "title": "OWASP IDE VulScanner",
+      "url": "https://destinjidee.com/owasp-ide-vulscanner",
```
Comment (kingthorin) [inline _data/tools.json:1940]: If it's an OWASP project why doesn't it have an actual OWASP project page?

File: _data/tools.json [hunk 2]
```diff
@@ -1934,5 +1934,14 @@
...
+      "owner": "DestinJiDee LTD",
+      "license": "Commercial or Free",
```
Comment (kingthorin) [inline _data/tools.json:1942]: If it's an OWADP project it can't be commercial

Comment (Jitu-Ranjan) [reply _data/tools.json]: Agreed and updated

File: _data/tools.json (hunk 1 above)
Comment (Jitu-Ranjan) [reply _data/tools.json]: Update the OWASP reference

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by Jitu-Ranjan** [comment]
@kingthorin I don't see the tool listed on the link below, do we need to run any workflow?
https://owasp.org/www-community/Vulnerability_Scanning_Tools

**Comment by kingthorin** [comment]
It's a SAST right so it'd be over here: https://owasp.org/www-community/Source_Code_Analysis_Tools

**Pull request merged**